index.html

```diff
@@ -350,66 +350,93 @@ <h3>Use Cases and Requirements</h3>
-<a>Verifiable credentials</a> are associated with <a>subjects</a>, not
-specific services; <a>holders</a> decide how to aggregate and manage
-<a>verifiable credentials</a> and the <a>verifiable presentations</a>
-associated with them.
+Acting as <a>Issuer</a>, <a>Holder</a>, or <a>Verifier</a> requires neither registration
+nor approval by any authority.
           </li>
           <li>
-<a>Subjects</a> can easily acquire and control their identifiers.
+<a>Verifiable Presentations</a>  allow any <a>Verifier</a> to verify the authenticity
+of claims from any Issuer.
           </li>
           <li>
-<a>Holders</a> control which <a>verifiable credentials</a> to use and when.
+<a>Holders</a> can receive <a>Verifiable Credentials</a> from anyone.
           </li>
           <li>
-<a>Holders</a> can freely choose and change the agents they employ to help
-them manage their <a>verifiable credentials</a> and generate and share
-<a>verifiable presentations</a>.
+<a>Holders</a> can interact with any <a>Issuer</a> and any <a>Verifier</a> through
+any User Agent.
           </li>
           <li>
-<a>Holders</a> can share <a>verifiable presentations</a> that can be verified
-without revealing the identity of the <a>verifier</a> to the <a>issuer</a>.
+<a>Holders</a> can store <a>Verifiable Credentials</a> in any location, without
+affecting their verifiability and without the <a>Issuer</a> knowing anything about
+where they are stored or when they are accessed.
           </li>
           <li>
-<a>Holders</a> in <a>presentations</a> can either disclose the attributes or
-satisfy <a>derived predicates</a> requested by the <a>verifier</a>.
-<a>Derived predicates</a> are boolean conditions like greater than, less than,
-equal to, is in set, and so on.
+<a>Holders</a> can present <a>Verifiable Presentations</a>  to any <a>Verifier</a>
+without affecting authenticity of the claims and without revealing that action
+to the Issuer.
           </li>
           <li>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> are
-expressed in one or more standard, machine-readable data formats, which can
-also be extended with minimal coordination.
+A <a>Verifier</a> can verify <a>Verifiable Presentations</a>  from any <a>Holder</a>,
+containing proofs of claims from any Issuer.
           </li>
           <li>
-<a>Issuers</a> issue <a>verifiable credentials</a>, and <a>holders</a> store
-them. <a>Holders</a> offer <a>verifiable presentations</a>, and
-<a>verifiers</a> verify them. Issuing of <a>credentials</a>, storage of
-<a>credentials</a>, generation of <a>presentations</a>, sharing of
-<a>presentations</a>, and verification of <a>presentations</a> are
-each independent processes.
+Verification can't depend on direct interactions between <a>Issuers</a> and <a>Verifiers</a> . 
           </li>
           <li>
-<a>Verifiable credentials</a> can be revoked by the <a>issuer</a>.
+Verification can't reveal the identity of the <a>Verifier</a> to any Issuer. 
```

Comment:
Again, not always true.

Reply:
perhaps this should say shouldn't instead of can't